LED_data_scrapper/spiders/products_spider.py

```diff
@@ -0,0 +1,51 @@
+""" Module to scrap the data of all the LEDs fromgiven website """
+import scrapy
+from scrapy.loader import ItemLoader
+from LED_data_scrapper.items import ProductItem
+
+class ProductsSpider(scrapy.Spider):
+    """
+    Spider class for Product, here only scrapping LED data
+    """
+
+    name = "products"
+
+    start_urls = ['https://pakistanistores.com/prices/home-appliances/led-tv-prices', ]
+
+    def parse(self, response):
+        # pylint: disable=arguments-differ
+        """ Parsing function """
+        products = response.xpath('//ul[@class="row search-ul"]/\
```

Comment:
This XPath should be improved, as the HTML structure is likely to change and CSS classes are not that likely to change. So for this example, you can just grep it using the `product` CSS class.

Please review and improve all Xpaths similar to the above example.